crates/cli-support/src/js/mod.rs

```diff
@@ -194,9 +198,10 @@ impl<'a> Context<'a> {
         &mut self,
         module_name: &str,
         needs_manual_start: bool,
-    ) -> Result<(String, String), Error> {
+    ) -> Result<(String, String, Option<String>), Error> {
```

Comment:
This return type is getting unfortunately pretty unwieldy, could this be updated to a named structure with documented fields?